README.md

```diff
@@ -116,31 +113,37 @@ In order to run fluentd as a docker image to send the log, siem and metrics data
 
 ```text
 
-For Newrelic as the observability platform, execute these commands to setup the docker container running the fluentd installation
+For NewRelic as the observability platform, execute these commands to setup the docker container running the fluentd installation
 
 1. Execute 'docker build --build-arg SOURCE="JFRT" --build-arg TARGET="NEWRELIC" -t <image_name> .'
 
-Command example
+    Command example
+
+    'docker build --build-arg SOURCE="JFRT" --build-arg TARGET="NEWRELIC" -t jfrog/fluentd-newrelic-rt .'
 
-'docker build --build-arg SOURCE="JFRT" --build-arg TARGET="NEWRELIC" -t jfrog/fluentd-newrelic-rt .'
+    The above command will build the docker image.
 
-The above command will build the docker image.
+2. Fill the necessary information in the Dockerenvfile.txt file
```

Comment:
It's common to use `.env` for env file. Not a blocker but we should follow common practice.